client/src/i18n/en/stock.json

```diff
@@ -89,7 +89,8 @@
       "DOWNLOAD_TEMPLATE_HERE" : "Download the template here",
       "DESCRIPTION" : "Please download and then fill in the template file appropriately with your stock information. Then select the completed file via the input box below",
       "PLEASE_SELECT_DEPOT" : "Please select a depot first",
-      "UPLOAD_SUCCESS" : "Stock imported successfully"
+      "UPLOAD_SUCCESS" : "Stock imported successfully",
+      "UPLOAD_DATE_REQUIRED" : "Define the importation date please"
```

Comment:
In English, putting the "please" at the end makes it sound a lot less formal.  To make this more formal, just switch the "please" to the beginning: "Please define the importation date".